partcad/src/partcad/project.py

```
@@ -1165,6 +1171,21 @@ def _add_component(
 
         return True
 
+    def add_sketch(self, kind: str, path: str, config={}) -> bool:
```

Ruff (0.8.2)

Missing type annotation for function argument `config`

(ANN001)